components/dashboard/src/user-settings/Preferences.tsx

```diff
@@ -81,14 +81,14 @@ export default function Preferences() {
         [toast, setUser, workspaceTimeout, billingMode],
     );
 
-    const clearAutostartWorkspaceOptions = useCallback(async () => {
+    const clearCreateWorkspaceOptions = useCallback(async () => {
         if (!user) {
             return;
         }
         AdditionalUserData.set(user, { workspaceAutostartOptions: [] });
         setUser(user);
         await getGitpodService().server.updateLoggedInUser(user);
-        toast("Your autostart options were cleared.");
+        toast("Remembered workspace options have been cleared.");
```

Comment:
nit: "Remembered" sounds weird to me. I'd go with:
```suggestion
        toast("Workspace options have been cleared.");
```

Reply:
Or "Last used options [...]", which would match the sub-heading.